Add pitchme_to_readme make target

denstorti opened the pull request:
## Summary
- Add pitchme_to_readme make target
   - Example:
    FOLDER_NAME=classes/01class/git make pitchme_to_readme
- It will create a README.md file in the same FOLDER_NAME removing all snippets starting with `@`.

- Preview: https://gitpitch.com/devopsacademyau/academy/Add_script_to_convert_pitchme_to_readme?p=classes/01class/git#/

caiocezart commented:
Can you create a folder classes/01class/exercises/c01-e05 and add the exercises part of the content to the README.md there?

Also update https://github.com/devopsacademyau/academy/blob/master/classes/01class/exercises/README.md to link to your exercises on that folder

denstorti commented:
- Created separate folders for git exercises.
  - Adopted the new pattern for exercise ID, including the content type. 
  - Example: `c01-git-01`.
- Updated Exercises README.md for class 01.
- Removed homework exercises from PITCHME.md and README.md file

Ready to be reviewed again @caiocezart
